app/src/org/commcare/activities/EntitySelectActivity.java

```diff
@@ -664,9 +669,16 @@ private void handleSearchStringCallout(Intent intent) {
         }
     }
 
-    private void handleFingerprintMatchCallout(Intent intent) {
-        OrderedHashtable<String, String> guidToMatchConfidenceMap =
-                SimprintsCalloutProcessing.getConfidenceMatchesFromCalloutResponse(intent);
+    private void handleFingerprintMatchCallout(
+            Intent intent,
+            @IdentityCalloutHandler.IdentityProvider String identityProvider) {

```

Comment:
This is just a more memory-efficient replacement for Enums on Android. [This](https://guides.codepath.com/android/Replacing-Enums-with-Enumerated-Annotations) is a quick gide on this in case you are interested.